### DISCUSSION TITLE
Yim Safe for online?

### DISCUSSION THREAD

**Discussion by ZippingBoat9588** [Mr-X-GTA/YimMenu#13] [Q&A]
After the recent banwave this yim menu is updated a few Times so is the Bypass undetected or deteced?

**Comment by L7NEG** [comment]
> After the recent banwave this yim menu is updated a few Times so is the Bypass undetected or deteced?

https://github.com/Mr-X-GTA/YimMenu/discussions/12#discussioncomment-10801222

**Reply by ZippingBoat9588** [reply]
What does that now mean? Not good at understanding Code

**Reply by L7NEG** [reply]
In simple this yimmenu have hooked the battleye state like its faking that you are legit be enabled player so it is safe

**Reply by ZippingBoat9588** [reply]
Alright is there an discord for this Version of yim for better Update notification?

**Reply by L7NEG** [reply]
nope but you can follow yimura's matrix chat you can find it in the note here https://github.com/UltimateMenu/UltimateMenu

**Comment by Tessio** [comment]
To ask or say that it is “Safe” is contradictory.

“It is safe” is a very ambiguous word.

The situation is that it has not yet been determined exactly what caused the wave of bans, since it was not against a specific mod menu or against a specific recovery method, even there were legitimate people who never used mod menu that got banned in this banwave.

What some of us have in mind is that it was because the users had never opened GTA Online with BattlEye activated and that Arxan flagged the accounts or it is completely detectable that you play without BattlEye activated and using the Bypass to be able to enter, but to be sure it takes some time.

Note
Since the banwave happened, I kept using mod menus, but I did the thing of playing at least 30 minutes with BattlEye activated in the Online and it's been 8 days now, my 3 accounts are still alive.
